scripts/catalog-to-astro-content-directory.js

```diff
@@ -7,10 +7,17 @@ import os from 'node:os';
 const __filename = fileURLToPath(import.meta.url);
 const scriptsDir = path.dirname(__filename);
 
+const cleanRelativePath = (relativePath, type) => {
+  const position = relativePath.indexOf(`${type}${path.sep}`);
```

Comment:
Take the following path `/root/something-events/other/events/my-event/index.md` and type equals `events`. The `indexOf` will match the first `events/` which is not what we expect. 

As pointed [here](https://github.com/event-catalog/eventcatalog/pull/812#issuecomment-2356743840) add `path.sep` before the type too.

Reply:
I added. Please review again.

Reply:
Bump, @carlosallexandre